build.sbt

```diff
@@ -5,9 +5,6 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 val libName = "play-conditional-form-mapping"
 
 val compileDependencies = PlayCrossCompilation.dependencies(
-  play25 = Seq(
```

Comment:
I think we would still want to have this library cross-compiled for play 2.5. It's a public repo which is used by developers outside of HMRC

Reply:
the recommended version of "sbt-play-cross-compilation" (2.0.0) no longer has support for play 2.5. Thats. why I had to remove it

Reply:
OK, that should be fine then, people will simply have to keep downloading the older 2.5 compatible version... we stop publishing a 2.5 copy. We don't actually actively make any additions to it anyways.